include/rmm/device_scalar.hpp

```diff
@@ -92,6 +91,14 @@ class device_scalar {
 
  private:
   rmm::device_buffer buff{sizeof(T)};
+
+  inline void _memcpy(void *dst, const void *src, size_t count,
```

Comment:
This class only holds a single object. so you don't seem to need `count`.  But even if you do, since this method is private to scalar, would it ever take anything other than `sizeof(T)`?

Reply:
Most likely not. Good catch.